src/middleware/__tests__/batch.test.js

```diff
@@ -63,7 +101,7 @@ describe('batchMiddleware', () => {
       matcher: '/graphql/batch',
       response: {
         status: 200,
-        body: [{ data: {} }, { id: 2, data: { ok: 2 } }],
```

Comment:
Changed to preserve the original error message - Otherwise it will hit the new path, because a response does not have an id. Alternatively I can change it to return an ID that has not been requested